pkgs/development/python-modules/pgpy/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cryptography
+, pyasn1
+, six
+, singledispatch
+}:
+
+buildPythonPackage rec {
+  pname = "PGPy";
+  version = "0.4.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "04412dddd6882ac0c5d5daf4326c28d481421851a68e25e7ac8e06cc9dc2b902";
+  };
+
+  propogatedBuildInputs = [ cryptography pyasn1 six singledispatch ];
```

See https://github.com/SecurityInnovation/PGPy/blob/f1c3d68e32c334f5aa14c34580925e97f17f4fde/setup.py#L36
> only depend on enum34 if Python is older than 3.4

We can constrain the addition of that with
```suggestion
  propogatedBuildInputs = [ cryptography pyasn1 six singledispatch ] ++ lib.optional (pythonOlder "3.4") enum34;
```